cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -321,54 +315,9 @@ data:
-
-      {{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostDeployment) (.Values.kubecostAggregator) .Values.kubecostAggregator.enabled }}
-      {{- fail "Query Service Replicas should not be used at the same time as the Kubecost Aggregator" }}
-      {{- end }}
-
-        location /model/allocation {
-            proxy_connect_timeout       {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_send_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_read_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_pass http://queryservice/allocation;
-            proxy_redirect off;
-            proxy_http_version 1.1;
-            proxy_set_header Connection "";
-            proxy_set_header  X-Real-IP  $remote_addr;
-            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
-        }
-
-        location /model/assets {
-            proxy_connect_timeout       {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_send_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_read_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_pass http://queryservice/assets;
-            proxy_redirect off;
-            proxy_http_version 1.1;
-            proxy_set_header Connection "";
-            proxy_set_header  X-Real-IP  $remote_addr;
-            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
-        }
-
-        # to get memory profile from query service need to prefix all request by queryservice/
-        # for example if you want heap dump from query service end point should be
-        # /model/queryservice/debug/pprof/heap to get queryservice heap dumps
-        location ~ /model/queryservice/(.*)$ {
-            proxy_connect_timeout       {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_send_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_read_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 600 }};
-            proxy_pass http://queryservice/$1;
-            proxy_redirect off;
-            proxy_http_version 1.1;
-            proxy_set_header Connection "";
-            proxy_set_header  X-Real-IP  $remote_addr;
-            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
-        }
-    {{- end }}
-
-{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostDeployment) (.Values.kubecostAggregator) .Values.kubecostAggregator.enabled }}
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
+    # TODO make aggregator route the default, start special-casing
```

Comment:
we might want to hold off on default route for aggregator, until we confirm all apis are moved over

Reply:
Agreed, I believe the current state is still special-casing aggregator.